.github/actions/rust/action.yml

```diff
@@ -22,7 +22,7 @@ runs:
   using: composite
   steps:
     - name: Install Rust
-      uses: dtolnay/rust-toolchain@21dc36fb71dd22e3317045c0c31a3f4249868b17 # master
+      uses: dtolnay/rust-toolchain@master # zizmor: ignore[unpinned-uses]
```

Comment:
Since I had a lot of time to wait for the CI runs during debugging, I noticed some unrelated issues and fixed them. This is one of them.

Reply:
Dependabot doesn't update SHAs that aren't associated with tags, and `dtolnay/rust-toolchain` doesn't tag any releases, so we were stuck on an old version.

Reply:
Preference for a hash, even if it idoesn't auto-update. In my eyes, the more deterministic our CI is the better.